### ISSUE TITLE
Add a color picker to choose a color

### ISSUE THREAD

**Issue by NathanS-8500** [Peppie84/FS22_HudColor#9]
I have no issues with the mod, but would it be possible to add support for the FS22_colorConfigurator? Still maintaining the savegame restart for the selected color change. I use this a lot and I think it would be nice to have this option also.

**Comment by Peppie84** [comment]
Hi @NathanS-8500 , what do you mean with "to add support for the FS22_colorConfigurator" ? Both mods are different. Do you mean, to integrate a color picker to select a color?

**Comment by NathanS-8500** [comment]
@Peppie84 Yes. Integrating a color picker would give the user a little more option on their choice of colors.
